src/data/__tests__/cookingTimes.test.js

```diff
@@ -1,6 +1,6 @@
 /* global test, describe, it, expect */
 'use strict'
-import cooking-times from '~/GroceriStar/cd/cooking-times.js'
+import cooking-times from '../cd/cooking-times.js'
```

Comment:
this path doesn't right. I have showed you in comment like this
```
import dash from '../../../dash'
```
because this repo haven't folder cd. this ../../../ give access to files that located in root of repo